Agilefreaks/FreaksCatalog-API
# [118] add role to a freak

## Description
 In this PR we can add a freak with a role

## Related

[[42] [Add Role to a Freak](https://trello.com/c/1NIOAZss/118-add-role-to-a-freak)

## Steps to Test or Reproduce
Go in insomnia and create a Freak.You can add this mutation:
```
mutation {
	freakCreate(
		firstName: "john",
		lastName: "doe",
		description: "ceva",
		email: "freak@gmail.com",
		roleId: 1
	) {
		firstName
		lastName
		description
		email
		role {
			name
		}
	}
}
```
Query freaks to see that the new freak was create:
```
query Freaks($last: Int, $before: String) {
    freaks(last: $last, before: $before) {
    
        edges {
            node {
                name
                firstName
                lastName
                description
                role {
                    name
                }       
            }
        }
    }
}
```
